docs/quickstart/self-managed/docker.md

```diff
@@ -3,6 +3,7 @@ title: Docker
 layout: default
 nav_order: 3
 parent: Self-managed
+nav_exclude: true
```

Comment:
Won't this create a 404 for the link from https://elastic.github.io/opentelemetry/quickstart/ ?

Reply:
it won't, it will just not appear in the navigation